apps/src/applab/commands.js

```diff
@@ -1381,7 +1384,9 @@ applabCommands.startWebRequest = function (opts) {
   apiValidateType(opts, 'startWebRequest', 'callback', opts.func, 'function');
   var req = new XMLHttpRequest();
   req.onreadystatechange = applabCommands.onHttpRequestEvent.bind(req, opts);
-  req.open('GET', opts.url, true);
+  var url = XHR_PROXY_PATH + '?u=' + encodeURIComponent(opts.url.replace(/ /g, '%20')) +
+      '&c=' + Applab.channelId;
```

Comment:
Done. I don't think we'd ever get this far with a null `Applab.channel_id`, but I've added a check to enforce this just in case.